### PR TITLE
[DOCFIX] add the chinese version of DebuggingGuide.md

### PR DESCRIPTION
**Pull request by zhutongxue**

### CODE REVIEW

**Comment by ifcharming** [comment]
ok to test

File: docs/cn/Debugging-Guide.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+title: 调试指南
+group: Resources
+---
+
+* Table of Contents
+{:toc}
+
+本页主要是关于Alluxio使用过程中的一些指导和提示，方便用户能够更快的解决使用过程遇到的问题。
+注意: 本页并不包含Alluxio使用过程中遇到的所有问题。
+用户可随时向 [Alluxio邮件列表](https://groups.google.com/forum/#!forum/alluxio-users)提交列表。
```
Comment (ifcharming) [inline docs/cn/Debugging-Guide.md]: s/提交列表/提交问题

File: docs/cn/Debugging-Guide.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+## Alluxio安装常见问题解答
+
+#### 问题: 在本地机器上初次安装使用Alluxio失败，应该怎么办？
+
+解决办法: 首先检查目录`{ALLUXIO_HOME}/logs`下是否存在master和worker日志，然后按照日志提示的错误信息进行操作。否则，再次检查是否遗漏了[本地运行Alluxio](Running-Alluxio-Locally.html)里的配置步骤
+
+典型问题:
+
+- `ALLUXIO_UNDERFS_ADDRESS`配置不正确
+- 如果 `ssh localhost` 失败, 确定`~/.ssh/authorized_keys`文件中包含主机的ssh公钥
```
Comment (ifcharming) [inline docs/cn/Debugging-Guide.md]: 请确认`~/.ssh/authorized_keys`文件中包含主机的ssh公钥

File: docs/cn/Debugging-Guide.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+#### 问题: 打算在Spark/HDFS集群中部署Alluxio，有什么建议？
+
+解决办法: 按照[集群环境运行Alluxio](Running-Alluxio-on-a-Cluster.html), 
+ [Alluxio配置HDFS](Configuring-Alluxio-with-HDFS.html)提示操作。
+
+提示:
+
+- 通常情况下, 当Alluxio workers和计算框架的节点部署在一起的时候，性能可达到最优
+- 如果你正在使用Mesos或者Yarn管理集群,也可以将Mesos和Yarn集成到Alluxio中，使用Mesos和Yarn可方便集群管理
+- 如果底层存储是远程的，比如说S3或者远程HDFS,这种情况下，使用Alluxio是最合适的
```
Comment (ifcharming) [inline docs/cn/Debugging-Guide.md]: 使用Alluxio会非常有帮助。

File: docs/cn/Debugging-Guide.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+## Alluxio性能常见问题解答
+
+#### 问题: 在Alluxio/Spark上进行测试（对大小为GBs的文件运行单词统计），相对于HDFS/Spark，性能并无明显差异。为什么?
+
+解决办法: Alluxio通过使用分布式的内存存储（以及分层存储）和时间或空间的本地化来实现性能加速。如果数据集没有任何本地化, 性能加速效果并不明显。 
+
+## 环境
+
+Alluxio在不同的生产环境下可配置不同的运行模式。
+请确定当前Alluxio版本是最新的，可提供支持的。
```
Comment (ifcharming) [inline docs/cn/Debugging-Guide.md]: 版本是最新的并且是支持的版本。

**Comment by ifcharming** [comment]
Thanks a lot for the translation!
Overall looks great, I left a few comments.

**Comment by zhutongxue** [comment]
@ifcharming 
Thanks very much for your comments!
I have modified the Debugging-guid.md and pushed according to your suggestion.

**Comment by yupeng9** [comment]
LGTM

**Pull request merged**